src/NServiceBus.Core/Features/FeatureConfigurationContext.cs

```diff
@@ -41,6 +42,8 @@ internal FeatureConfigurationContext(ReadOnlySettings settings, IConfigureCompon
 
         internal RoutingComponent Routing { get; }
 
+        internal RecoverabilityComponent Recoverability { get; }
```

Comment:
Is this used anywhere?

Reply:
yes here for the acceptance testing framework: https://github.com/Particular/NServiceBus/pull/5503/files#diff-e6d8bf51552c83a965bcd54d6f1ea308L26

Reply:
I actually have some ideas how we could avoid this. I'll raise a separate PR though.